README.md

```diff
@@ -262,29 +262,29 @@ Please, have a look at the [`witnet/truffle-box`](https://github.com/witnet/truf
+|  WitnetRequestBoardTrustableDefault  ·  initialize     ·          -  ·          -  ·      75077  ·          30  ·          -  │
 ·······································|·················|·············|·············|·············|··············|··············
-|  WitnetRequestBoardTrustableDefault  ·  postRequest    ·     144525  ·     196359  ·     164155  ·          33  ·          -  │
+|  WitnetRequestBoardTrustableDefault  ·  postRequest    ·     144650  ·     196484  ·     164280  ·          33  ·          -  │
 ·······································|·················|·············|·············|·············|··············|··············
-|  WitnetRequestBoardTrustableDefault  ·  reportResult   ·     119164  ·     121313  ·     119760  ·          18  ·          -  │
+|  WitnetRequestBoardTrustableDefault  ·  reportResult   ·     119210  ·     121359  ·     119806  ·          18  ·          -  │
 ·······································|·················|·············|·············|·············|··············|··············
-|  WitnetRequestBoardTrustableDefault  ·  upgradeReward  ·      31319  ·      36462  ·      34748  ·           6  ·          -  │
+|  WitnetRequestBoardTrustableDefault  ·  upgradeReward  ·      31341  ·      36484  ·      34770  ·           6  ·          -  │
 ·······································|·················|·············|·············|·············|··············|··············
 |  Deployments                                           ·                                         ·  % of limit  ·             │
 ·························································|·············|·············|·············|··············|··············
 |  WitnetDecoderLib                                      ·          -  ·          -  ·    1930600  ·      28.7 %  ·          -  │
 ·························································|·············|·············|·············|··············|··············
-|  WitnetParserLib                                       ·          -  ·          -  ·    2570010  ·      38.3 %  ·          -  │
+|  WitnetParserLib                                       ·          -  ·          -  ·    2594996  ·      38.6 %  ·          -  │
 ·························································|·············|·············|·············|··············|··············
 |  WitnetProxy                                           ·          -  ·          -  ·     587730  ·       8.7 %  ·          -  │
 ·························································|·············|·············|·············|··············|··············
-|  WitnetRequestBoardTrustableDefault                    ·          -  ·          -  ·    2487604  ·        37 %  ·          -  │
+|  WitnetRequestBoardTrustableDefault                    ·          -  ·          -  ·    3699603  ·      55.1 %  ·          -  │
```

Comment:
How did this become +49% more expensive? :thinking:

Reply:
Will double-check, but should mainly be because of:

- new getter methods declared in `IWitnetRequestBoardView`
- new getter methods declared in `IWitnetRequestParser`
- new methods in `IWitnetRequestBoardAdminACLs`

Reply:
Cool. It's deployment cost anyway, so not a big concern. Thanks! :smiley: